src/main/java/org/cru/aemscraper/model/PageData.java

```diff
@@ -6,6 +6,11 @@
 import java.util.Objects;
 
 public class PageData {
+
+    public enum CHANGE_TYPE {
+        Activate
```

Comment:
This is the only one because the scraper is only used for importing create/update data?

Reply:
Correct.